### PR TITLE
Prometheus endpoint fixes

### PR DESCRIPTION
**Pull request by vdimir**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Non-significant (changelog entry is not needed)

Fixes for Prometheus endpoint:
- Replace dots with underscores in metric names.
- Async metrics updated at program startup. It is the reason why we  don't catch incorrect AsyncMetrics names with dots in test.
-  Change default port to `9363`. I have grab free port on Prometheus [wiki page](https://github.com/prometheus/prometheus/wiki/Default-port-allocations) as stated in [docs](https://prometheus.io/docs/instrumenting/writing_exporters/#port-numbers). (It don't require any approve from maintainers or additional permissions)

Also I have edit link on _Exporters and integrations_ a page (on [review](https://github.com/prometheus/docs/pull/1513))

Ref #7369

### CODE REVIEW

**Review by alexey-milovidov — APPROVED** [review]

**Comment by alexey-milovidov** [comment]
> Unit tests — fail: 2

Glitch while interacting with ZooKeeper in CI environment.

> Functional stateless tests (release, processors) — fail:1

Wasn't introduced in this PR.

> Description check — Category section not found 

Already Ok.

File: dbms/programs/server/PrometheusMetricsWriter.cpp [hunk 1]
```diff
@@ -20,6 +22,11 @@ void writeOutLine(DB::WriteBuffer & wb, T && val, TArgs &&... args)
     writeOutLine(wb, std::forward<TArgs>(args)...);
 }
 
+void replaceInvalidChars(std::string & metric_name)
```
Comment (azat) [inline dbms/programs/server/PrometheusMetricsWriter.cpp]: FWIW: normalize looks better

Comment (alexey-milovidov) [reply dbms/programs/server/PrometheusMetricsWriter.cpp]: As long as it is a function in .cpp file with hidden linkage, I don't care too much :)

**Comment by vitlibar** [comment]
There is no need to backport it to 19.16, 19.17, 19.18 because [19.18](https://github.com/ClickHouse/ClickHouse/tree/19.18/dbms/programs/server) has no support for Prometheus endpoint.

**Pull request merged**